test/powershell/Language/Scripting/Requires.Tests.ps1

```diff
@@ -27,4 +27,14 @@ Describe "Requires tests" -Tags "CI" {
         }
     }
 
-}
\ No newline at end of file
+    Context "Interactive requires" {
+        It "Successfully does nothing when given '#requires' interactively" {
+            $settings = [System.Management.Automation.PSInvocationSettings]::new()
+            $settings.AddToHistory = $true
+
+            $ps = [powershell]::Create()
+            $ps.AddScript("#requires`n10")
+            $ps.Invoke(@(), $settings) | Should -BeExactly 10
```

Comment:
Please dispose `$ps` after you done with it.
Also, I think it's not necessary to use `PSInvocationSettings` in this test. Just `$ps = [powershell]::Create(); $ps.AddScript("#requires`n10").Invoke() | Should -BeExactly 10` will do.

Reply:
I see, we need to hit the $^/$$ setting code. Never mind.